cloud/pkg/cloudhub/servers/wsserver/server.go

```diff
@@ -15,11 +15,12 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/handler"
 
 	"github.com/gorilla/mux"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model"
```

Comment:
please seperate vendor and kubeedge packages.

Reply:
Done.